packages/botonic-plugin-flow-builder/src/webview/contents-context.ts

```diff
@@ -0,0 +1,11 @@
+import { createContext } from 'react'
+
+import { WebviewContentsContextType } from './types'
+
+export const WebviewContentsContext = createContext<WebviewContentsContextType>(
+  {
+    getTextContent: () => undefined,
+    getImageSrc: () => undefined,
+    setCurrentLocale: () => undefined,
+  }
+)
```

Comment:
Doubt: Shouldn't this context be typed as `UseWebviewContents`? This should also contain the `isLoading` attribute, right?

Reply:
IMO the context does not need the isLoading, because the hook will only be called 1 time in the index of the webview and the context will be built. 
Then in each component inside the webview you can use the context functions to get each content

Reply:
But what happens if it takes longer than expected in loading the contents and you want to display a loading animation meanwhile? If you don't return this attribute the main app won't know it and won't be able to know when the contents have been loaded, am I right?